BlueprintUI/Sources/Blueprint View/ElementPreview.swift

```diff
@@ -251,43 +251,89 @@ extension ElementPreview {
 /// Via https://developer.apple.com/documentation/swiftui/securefield/3289399-previewdevice
 @available(iOS 13.0, *)
 public extension PreviewDevice {
+    
+    /// iPhone 7
+    
     static var iPhone7 = PreviewDevice("iPhone 7")
     static var iPhone7Plus = PreviewDevice("iPhone 7 Plus")
     
+    /// iPhone 8
+    
     static var iPhone8 = PreviewDevice("iPhone 8")
     static var iPhone8Plus = PreviewDevice("iPhone 8 Plus")
     
-    static var iPhoneSE = PreviewDevice("iPhone SE")
+    /// iPhone SE
+    
+    static var iPhoneSE_1 : PreviewDevice {
+        if isAtLeast(13, 4, 1) {
+            return PreviewDevice("iPhone SE (1st generation)")
+        } else {
+            return PreviewDevice("iPhone SE")
+        }
+    }
+    
+    static var iPhoneSE_2 : PreviewDevice {
```

Comment:
Should we mark this one `@available(iOS 13.4.1)` instead of making it possible for it to be 2 different devices?

Reply:
:thumbsup: